src/handbook/company/board.md

```diff
@@ -19,3 +19,36 @@ Two weeks before the next board meeting
 1. Set the strategic topics to discuss. Propose 3 topics to attendees and ask them to pick two
 1. Set agenda structure and send the concept agenda to attendees
 
+## Board charter  
+
+### Purpose of the charter
+
+The Board Charter sets out the role, composition, and responsibilities of the Board of Directors.
+
+### Purpose of the board
+
+The Board has two broad purposes, compliance and performance. Making sure the company is compliant with rules and regulations from governing institutions. In addition they must be up-to-date and hold the company accountable for how they conduct their operations and relaying information to the public.
+
+### Roles and Responsibilities
+
+See [Roles and Responsibilities](./security/information-security-roles.md#board-of-directors)
+
+### Membership and Term
+
+...
```

Comment:
Members are Nick, ZJ, Betty Ma (Open Core Ventures), and Adit Singh (Cota Capital)

Reply:
Term is undefined